### ISSUE TITLE
Replace shortstrings with `ByteArray`

### ISSUE THREAD

**Issue by piotmag769** [foundry-rs/starknet-foundry#1507]
Currently related to:
~~- `declare`~~ (separate issue)
- `env`
- `fs` module (imo all listed things should be of type `ByteArray`):
  - path to file
  - strings in JSON
  - strings in txt:
    - single quoted strings
    - double quoted strings
 - `should_panic`

**Comment by Arcticae** [comment]
Also `declare`  i think should be called with `ByteArray`

**Comment by Arcticae** [comment]
We should consider other native solutions with `declare`, like providing the full path to the contract module, and possibly referring to a generated constant (like a `NAME` for example)

**Comment by Arcticae** [comment]
Blocked by #1531
